src/modules/simulation/gz_bridge/CMakeLists.txt

```diff
@@ -33,108 +33,105 @@
 
 # Find the gz_Transport library
 # Look for GZ Ionic or Harmonic
-find_package(gz-transport NAMES gz-transport14 gz-transport13)
```

Comment:
Other than failing the build, I think it would be more productive to provide a warning to the user that clearly states that these packages are missing and therefore gz-bridge will not be compiled. 

@Jaeyoung-Lim I guess you wanted this to be the case?